docs/bridge/architecture.mdx

```diff
@@ -36,18 +36,17 @@ The Connector Contracts handle the transfer of specific asset types:
 ### Archival Node
 The Archival Node stores all past transactions, typically spanning a longer time period (e.g., a month or week ago). It serves as a historical record of transaction data.
 
-### Indexer
-The Indexer fetches blockchain data and stores it in a database, allowing for efficient retrieval and querying of transaction information.
+### Events Indexer
+An Events indexer is a component that scans the blockchain and reads all events on it. If an event is a Calimero event it is published to the message queue. After that relayer and bridge service read the message queue and perform their job on those events
```

Comment:
Please add '.' at the end of the sentence

Reply:
Okay, i'll also update the image as well